mk/generate.mk

```diff
@@ -113,3 +115,7 @@ generate/api: protoc/common/v1alpha1 protoc/mesh protoc/mesh/v1alpha1 protoc/obs
 
 generate/test-server:
 	$(PROTOC_GO) test/server/grpc/api/*.proto
+
+.PHONY: generate/fix-embed
```

Comment:
Something that I did not foresee is that we would need another word-around for this approach :/ without this now controller-gen complains that there is no `schema.yaml` file - I'm guessing it imports a policy package and then the `//go:embed` match is triggered and causes this it to fail.